drivers/hyperv/powershell.go

```diff
@@ -38,12 +38,12 @@ func cmdOut(args ...string) (string, error) {
 	return stdout.String(), err
 }
 
-func cmd(args ...string) error {
+func Cmd(args ...string) error {
 	_, err := cmdOut(args...)
 	return err
 }
 
-func parseLines(stdout string) []string {
+func ParseLines(stdout string) []string {
```

Comment:
This feels like it should stay as an internal function: it's not particularly unique or interesting to other users.

Also, any exported function should have a comment describing what it is used for.